test/Conversion/tritongpu_to_llvm_ampere.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: triton-opt %s -split-input-file --allocate-shared-memory --convert-triton-gpu-to-llvm=compute-capability=80 2>&1 | FileCheck %s
```

Comment:
That should go in test/Conversion/tritongpu_to_llvm.mlir

Reply:
Moved it to  test/Conversion/tritongpu_to_llvm.mlir